contracts/op-finality-gadget/src/utils.rs

```diff
@@ -53,29 +49,3 @@ pub fn query_finality_provider(
 
     Ok(res)
 }
-
-/// TODO: query_grpc need to be replaced with cosmwasm::std::QuerierWrapper.query_grpc
```

Comment:
to provide context, this is not needed anymore b/c the dependency has been upgraded